src/qt/walletframe.cpp

```diff
@@ -3,20 +3,19 @@
 // file COPYING or http://www.opensource.org/licenses/mit-license.php.
 
 #include <qt/walletframe.h>
-#include <qt/walletmodel.h>
 
-#include <qt/bitcoingui.h>
+#include <qt/walletmodel.h>
 #include <qt/walletview.h>
 
 #include <cassert>
 
 #include <QHBoxLayout>
 #include <QLabel>
 
-WalletFrame::WalletFrame(const PlatformStyle *_platformStyle, BitcoinGUI *_gui) :
-    QFrame(_gui),
-    gui(_gui),
-    platformStyle(_platformStyle)
+WalletFrame::WalletFrame(const PlatformStyle* _platformStyle, BitcoinGUI* _gui, QWidget* parent)
+    : QFrame(parent),
+      gui(_gui),
```

Comment:
1f913ac9a00b982c997e70603ba62eebbe4c9d03

nack style change without.

Reply:
Shouldn't a contributor consistently apply [`clang-format-diff.py`](https://github.com/bitcoin/bitcoin/tree/master/contrib/devtools#clang-format-diffpy) to his own commits?

Reply:
I personally don't if it makes the diff larger than necessary.

Reply:
Probably irrelevant... I think `clang-format-diff.py` changes are fine.

Reply:
I think in general it's important to limit changes to only the code that is touched, so that it is easy to see the real changes.
But in this case, bah, an argument is added and some spaces are added to the other lines it's fine with me.

Reply:
See comment above https://github.com/bitcoin/bitcoin/pull/17500/files#r347153585